Oscil16.h

```diff
@@ -58,7 +58,7 @@ char2mozzi.py infilename outfilename tablename samplerate
 */
 //template <unsigned int NUM_TABLE_CELLS, unsigned int UPDATE_RATE, bool DITHER_PHASE=false>
 template <uint16_t NUM_TABLE_CELLS, uint16_t UPDATE_RATE>
-class Oscil12
+class Oscil16
```

Comment:
Yes, going to 16 bits makes more sense than 12 bits, _but_, again, please just split it out into a separate pull request. I think we can come up with a yet more generic solution, but that's simply a separate discussion, and should not hold up the extended Teensy 3.x and SAMD21 support.